hack/testing/logging.sh

```diff
@@ -112,37 +117,55 @@ trap "cleanup" EXIT
 
 echo "[INFO] Starting logging tests at " `date`
 
-os::util::ensure::iptables_privileges_exist
+#os::util::ensure::iptables_privileges_exist
 # override LOG_DIR and ARTIFACTS_DIR
 export LOG_DIR=${LOG_DIR:-${TMPDIR:-/tmp}/origin-aggregated-logging/logs}
 export ARTIFACT_DIR=${ARTIFACT_DIR:-${TMPDIR:-/tmp}/origin-aggregated-logging/artifacts}
 os::util::environment::use_sudo
-os::util::environment::setup_all_server_vars "origin-aggregated-logging/"
+#os::util::environment::setup_all_server_vars "origin-aggregated-logging/"
 
 os::log::system::start
 
+OCP_VERSION="v1.4.1"
 export KUBELET_HOST=$(hostname)
+export CLUSTER_UP_DIR=/tmp/origin-aggregated-logging/cluster
```

Comment:
Are you suggesting maybe I need a mkdir statement? It works on my AMI, but that could just be a fluke